manim/mobject/graph.py

```diff
@@ -382,6 +382,34 @@ def construct(self):
                 graph = Graph(list(G.nodes), list(G.edges), layout="partite", partitions=[[0, 1]])
                 self.play(Create(graph))
 
+    The representation of a linear artificial neural network is facilitated
+    by the use of the partite layout and defining partitions for each layer.
+
+    .. manim:: LinearNN
+        :save_last_frame:
+
+        class LinearNN(Scene):
+
+            def construct(self):
+                edges = []
+                partitions = []
+                c=0
+                layers = [2,3,3,2]         # the number of neurons on each layer
+
+                for i in layers:
+                    partitions.append([j for j in range(c+1, c+i+1)])
+                    c += i
+                for i, v in enumerate(layers[1:]):
+                        last = int(np.sum(layers[:i+1]))
+                        for j in range(v):
+                            for k in range(last-layers[i], last):
+                                edges.append((k+1, j+last+1))
+
+                vertices = np.arange(1, int(np.sum(layers))+1)
+
+                graph = Graph(vertices, edges, layout='partite', partitions=partitions, layout_scale=3, vertex_config={'radius':0.20})
```

Comment:
The line is a bit long and can be split:
```suggestion
                graph = Graph(
                    vertices,
                    edges,
                    layout='partite',
                    partitions=partitions,
                    layout_scale=3,
                    vertex_config={'radius': 0.20},
                )
```